lib/Evaluate/formatting.cpp

```diff
@@ -718,6 +718,43 @@ std::ostream &DescriptorInquiry::AsFortran(std::ostream &o) const {
   }
 }
 
+std::ostream &Assignment::AsFortran(std::ostream &o) const {
```

Comment:
I believe that we want to remove std::ostream from f18 to be more align with llvm, corret?